src/Components/Mood/Mood.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { observer } from "mobx-react";
+import styles from "./Mood.module.css";
+import type { MediaPresenter } from "../../MediaPresenter";
+import { templates } from "../../Template";
+import type { OutputPresenter } from "../../OutputPresenter";
+
+type Props = {
+  mediaPresenter: MediaPresenter;
+  outputPresenter: OutputPresenter;
+};
+
+/*
+/* Mood.tsx contains all mood options a user can pick
+/* following the media imported by the user
+*/
+
+export const Mood: React.FC<Props> = observer(
+  ({ mediaPresenter, outputPresenter }) => {
+    return (
+      <div className={styles.flexContainer}>
+        <p className={styles.optionHeading}>Length</p>
+        <div className={styles.optionContainer}>
```

Comment:
Looks like the options container could become its own component that could also live within this file. Same with the other options container below.